### DISCUSSION TITLE
URGENT! Wacatac.B!ml Trojan

### DISCUSSION THREAD

**Discussion by sheitmaster** [Rem0o/FanControl.Releases#1857] [General]
Fan control prompted me for an update today, it looked legit and windows defender stopped a Wacatac.B!ml Trojan when i pressed update.

**Comment by sheitmaster** [comment]
This trojan was in fact comming from FanControl, as it links back to the updater.exe file in my FanControl folder.

Make sure you don't have viruses and trojans on the system you devellop with, you can affect EVERYONE who's not up to date.

**Comment by Fr0stX76** [comment]
No need to panic and call out the develloper. Its a false positive typical of Windows defender. 

https://answers.microsoft.com/en-us/windows/forum/all/wacatac-false-positive-outbreak/0d92ef05-50db-4d12-92f4-fcfe8f0b966c

https://answers.microsoft.com/en-us/windows/forum/all/what-to-do-about-trojanwin32wacatacbml-false/d884df80-2dba-46c9-bd8b-4e2f93c9dd3f

**Comment by Rem0o** [comment]
Submited to Microsoft before publishing, normally it fixes fast, this time it takes a bit longer it seems like.

**Comment by sheitmaster** [comment]
Sorry.
I'm just really concious about security since i am working from home and i've never once had windows defender give me a false positive.
I was honestly really worried for a while there.
